portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,10 +60,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long timestamp = System.currentTimeMillis();
```

Comment:
Best to keep the units in the name when the type doesn't capture it.

`long timestampMillis = `